Templates/PHP/Model/EntityType.php.tt

```diff
@@ -140,9 +140,9 @@ if (property.Type.GetTypeString()[0] == '\\') { #>
     
     /**
     * Sets the <#=propertyName#>
-<# if (property.LongDescription != null) {
+<# if (property.LongDescription != null || property.Description != null) {
```

Comment:
Duplication of null Description/LongDescription check since it's being handled by GetSanitizedLongDescription